pkg/component/shoot/system/system.go

```diff
@@ -441,6 +441,13 @@ func (s *shootSystem) shootInfoData() map[string]string {
 }
 
 func (s *shootSystem) readOnlyRBACResources() []client.Object {
+	allowedSubResources := map[string]map[string][]string{
+		corev1.GroupName: {
+			"pods":  {"log"},
+			"nodes": {"proxy"},
```

Comment:
Hmmm. This doesn't only allow users of the viewer kubeconfig to use the read-only endpoints, but also the "admin" endpoints like exec, attach, etc.
Am I missing something?

Reply:
You're right. As discussed via Slack, I removed `nodes/proxy` again. Sorry @vlerenc, it seems you have to continue using an admin kubeconfig if you want to access the kubelet APIs.